composer/src/main/kotlin/com/gojuno/composer/Args.kt

```diff
@@ -87,23 +79,23 @@ data class Args(
                 required = false,
                 variableArity = true,
                 description = "Connected devices/emulators that will be used to run tests against. If not passed — tests will run on all connected devices/emulators. Specifying both `--devices` and `--device-pattern` will result in an error. Usage example: `--devices emulator-5554 emulator-5556`.",
-                order = 9
+                order = 8
         )
         var devices: List<String> = emptyList(),
 
         @Parameter(
                 names = arrayOf("--device-pattern"),
                 required = false,
                 description = "Connected devices/emulators that will be used to run tests against. If not passed — tests will run on all connected devices/emulators. Specifying both `--device-pattern` and `--devices` will result in an error. Usage example: `--device-pattern \"somePatterns\"`.",
-                order = 10
+                order = 9
         )
         var devicePattern: String = "",
 
         @Parameter(
                 names = arrayOf("--install-timeout"),
                 required = false,
                 description = "APK installation timeout in seconds. If not passed defaults to 120 seconds (2 minutes). Applicable to both test APK and APK under test.",
-                order = 11
+                order = 10
```

Comment:
I honestly don't remember why order is important here and certainly don't like that we need to maintain it manually..

Reply:
Its sorted alphabetically in `-help` without order specified =(